ref_weight_smooth: 40
ref_weight_path_length: 10
ref_weight_ref_deviation: 40

lower_bound: -0.5
upper_bound: 0.5

